packages/scheduler/lib/workers/monitor/monitor.worker.ts

```diff
@@ -7,7 +7,7 @@ import { setTimeout } from 'node:timers/promises';
 import type knex from 'knex';
 import { logger } from '../../utils/logger.js';
 
-interface MessageOut {
+interface ExpiredTasksMessage {
```

Comment:
unrelated but wanted to be consistent with the naming of the same notifying mechanism in the monitor and scheduling worker